packages/identity/src/__tests__/identity.spec.js

```diff
@@ -182,9 +220,16 @@ describe('Identity', () => {
 
   describe('create provisional', () => {
     let b64Identity;
+    let b64SmsIdentity;
 
     before(async () => {
-      b64Identity = await createProvisionalIdentity(trustchain.id, userEmail);
+      b64Identity = await createProvisionalIdentity(trustchain.id, 'email', userEmail);
+      b64SmsIdentity = await createProvisionalIdentity(trustchain.id, 'phone_number', userPhone);
+    });
+
+    it('cannot create a provisional with an invalid target', async () => {
+      // $FlowIgnore on ne peut pas tester une omelette sans casser des œufs
```

Comment:
english

Reply:
Fine! Stiffle my artistic expression, why don't you! =)

Fixed.